src/modules/todo-store.js

```diff
@@ -31,6 +31,39 @@ class TodoStore {
     this.#store.push(todo);
     this.#backup();
   }
+
```

Comment:
- [ ] **Optional:** There are also console errors that you should definitely consider fixing.

![image](https://user-images.githubusercontent.com/27709832/165912044-5f440533-4d06-4bcd-bcc0-479ff28177b2.png)

https://github.com/IshmaelKargbo/todo-list/blob/ca93a29b5aecca7131d2a95ad56f510f4f61d584/src/modules/todo-store.js#L19-L22

Reply:
thanks